app/assets/javascripts/Measure.js

```diff
@@ -14,27 +14,23 @@ const [Number, String, Boolean, Mixed, ObjectId, Date] = [
 
 const MeasureSchema = mongoose.Schema(
   {
-    // ID/other measure information
-    id: String,
-    measure_id: String,
+    // A version-specific UUID for the measure
     hqmf_id: String,
+    // A version-neutral UUID for the measure
     hqmf_set_id: String,
-    hqmf_version_number: Number,
+    // A Semantic Version-compliant string (e.g. "2.3.4") for the measure
+    hqmf_version_number: String,
+    // A CMS-style string (e.g. "CMS2v4") for the measure
     cms_id: String,
     title: String,
     description: String,
+    // EP/EH, or "unknown"
     type: String,
     category: { type: String, default: 'Uncategorized' },
 
     // Measure type variables
     episode_of_care: Boolean,
     continuous_constiable: Boolean,
```

Comment:
Huh? `continuous_variable`?

Reply:
That was fixed...not sure how it crept back in.